app/views/payouts/list.njk

```diff
@@ -16,12 +16,27 @@ Payments to your bank account - GOV.UK Pay
 
 {% block mainContent %}
 <div class="govuk-grid-column-full">
-  <h1 class="govuk-heading-l">Payments to your bank account</h1>
+  <h1 class="govuk-heading-l">
+    Payments to your bank account
+    <strong class="govuk-tag govuk-tag--grey">{{ "LIVE" if filterLiveAccounts else "TEST" }}</strong>
+  </h1>
 </div>
-<div class="govuk-grid-column-two-thirds">
-  <p class="govuk-body govuk-!-margin-bottom-8">Payments Stripe has made to your bank account.</p>
+<div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-8">
+  <p class="govuk-body">Payments Stripe has made to your bank account.</p>
+  {% if enableTestReports %}
+  <p class="govuk-body">
+    {% if filterLiveAccounts %}
+      <a class="govuk-link govuk-link--no-visited-state" href="{{ routes.formattedPathFor(routes.payouts.listStatusFilter, 'test') }}">Switch to test accounts</a>
```

Comment:
Can we only show `Switch to test accounts` if there are any test Stripe accounts ?

Reply:
We could do this as a custom property further up in the utility that checks permissions, do we think it's something that's needed though? Potentially showing the link helps keep the tools predictable and symmetrical as a reminder to teams that you can test this stuff with Stripe test accounts (when the content is added)

Reply:
Displaying link could indicate service has access to test reports which will never be the case. In case, if service has already got live Stripe account, it is very unlikely service would want to setup a test account (given service would already know about additional functionality that comes with Stripe).